### ISSUE TITLE
Is it possibale to add a warning lists?

### ISSUE THREAD

**Issue by ATang007ZH** [markusfisch/BinaryEye#471]
The warning lists is that needed  special attention.
When these barcode were scanned, show a warning message, like   Find a warning barcode!

**Comment by markusfisch** [comment]
Yes, this would be possible, but do you really want to maintain a list of contents on your device? Or did you mean to share this list in some way?

**Comment by Theta-Dev** [comment]
What do you mean with warning lists? Do you want the barcode scanner to warn the user of malicious URLs? 

I mean there are public lists of malicious sites like Google's Safebrowsing or the urlhaus malicious sites list that is included in uBlock Origin.

But most browsers already implement this feature, so why do you want to put it in a barcode scanner?
